mock.go

```diff
@@ -909,3 +913,150 @@ func assignRecords(m map[string]interface{}, record map[string]interface{}) erro
 
 	return nil
 }
+
+type mockContextKey string
+
+var errorInjectorContextKey mockContextKey = "failure_strategy"
```

Comment:
should we update this key text to match the change from 'failure strategy' to 'error injector'?